build-files/krona/2.8.1/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM ubuntu:jammy AS app
+
+ARG KRONA_VER="2.8.1"
+ARG KRAKENTOOLS_COMMIT="d4a2fbe2833c2392a724697c0c97a3014e748474"
+ARG BIOPYTHON_VER="1.84"
+ARG PANDAS_VER="2.2.3"
+
+
+USER root
+
+WORKDIR /
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="krona"
+LABEL software.version="${KRONA_VER}"
+LABEL description="Krona Tools is a set of scripts to create Krona charts from several Bioinformatics tools as well as from text and XML files."
+LABEL website="https://github.com/marbl/Krona"
+LABEL license="https://github.com/marbl/Krona/blob/master/KronaTools/LICENSE.txt"
+LABEL maintainer="Sage Wright"
+LABEL maintainer.email="sagemwright@gmail.com"
+
+# 'LABEL' instructions tag the image with metadata that might be important to the user
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    ca-certificates \
+    curl \
+    perl \
+    unzip \
+    python3 \
+    python-is-python3 \
+    python3-pip \
+    python3-setuptools \
+    make \
+    procps && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+ENV PATH="/root/miniconda3/bin:$PATH"
+
+RUN wget https://github.com/marbl/Krona/releases/download/v${KRONA_VER}/KronaTools-${KRONA_VER}.tar && \
+    tar -xvf KronaTools-${KRONA_VER}.tar && \
+    rm KronaTools-${KRONA_VER}.tar && \
+    mv KronaTools-${KRONA_VER} KronaTools && \
+    cd KronaTools && \
+    ./install.pl
+
+RUN pip install biopython==${BIOPYTHON_VER} \
+    pandas==${PANDAS_VER}
+
+RUN wget https://github.com/jenniferlu717/KrakenTools/archive/${KRAKENTOOLS_COMMIT}.zip && \
+    unzip ${KRAKENTOOLS_COMMIT}.zip && \
+    mv KrakenTools-${KRAKENTOOLS_COMMIT} KrakenTools && \
+    chmod +x KrakenTools/*
+
+ENV PATH=/KrakenTools:${PATH}
+
+WORKDIR /data
+
+CMD ["ktImportTaxonomy"]
+
+FROM app AS test
+
+RUN combine_kreports.py -h
+
+RUN extract_kraken_reads.py --help
+RUN ktImportTaxonomy
```

Comment:
These tests are great. They make sure that these scripts are in PATH and are executable. 

Do you know if there's some sample data somewhere that can used to make sure they run?

Reply:
yes, but it's a bit tricky since these run on the results of Kraken2. I can try to find some publicly available kraken outputs if you think that would be beneficial

Reply:
I think downloading an example file from Kraken2 for the multiqc module might be useful. MultiQC's test files are at https://github.com/MultiQC/test-data/tree/main/data/modules/kraken

Reply:
Thanks for the link to that repo! I didn't know that existed! change has been made